src/figureprinter/FigurePainter.java

```diff
@@ -63,7 +97,20 @@ public static void drawLeftBottomTriangle(int length){
      */
     public static void drawRightBottomTriangle(int length){
         // TODO : complete the method
+        for (int i = length; i >= 1; i--)
```

Comment:
for (int i = length; i >= 1; i--){

            for (int j = length; j > i; j--){
                System.out.print("  ");
            }
            for (int k = 1;k <= i;k++) {

                System.out.print(" *");
            }
            System.out.println("");
        }